src/lib/example.js

```diff
@@ -192,28 +198,35 @@ async function extractExample(example, name, lang) {
   const spin = ora(`${step}...`).start();
 
   try {
-    const src = 'github:o1-labs/zkapp-cli#main';
-
-    // Note: Extract will overwrite any existing dir's contents. That's ok here.
-    const TEMP = '.gittar-temp-dir';
-    await gittar.extract(src, TEMP, {
```

Comment:
It would be nice if `gittar.extract` accepted a local `src`, but I don't think it does (I wasn't able to get it to, at least).